### PR TITLE
Add STF renewal page & banner

### LINKED ISSUE #671: Add a page about STF
With the vote for the renewal of STF coming soon, it would be nice to have a page on our website that talks about the STF and how it supports us. Along with the extra page, creating a banner on our site that encourages people to vote would also help. ncberberi was able to get confirmation that we can advertise voting on STF to students.

For more information on what the STF is: [https://techfund.berkeley.edu/](https://techfund.berkeley.edu/)

Voting will take place during the ASUC elections on April 5-7

### PR DESCRIPTION
**Pull request by singingtelegram**
Fixes #671.

### CODE REVIEW

**Comment by singingtelegram** [comment]
Interestingly, STF's own website seems to have some permission issues: https://techfund.berkeley.edu/current-status-student-technology-fund

File: ocfweb/templates/base.html [hunk 1]
```diff
@@ -97,30 +97,17 @@
...
-                            {% endif %}
-                            {{current_lab_hours|lab_hours_time}}
-                            on {% now "l" %} {% lab_hours_holiday holidays %}
-                            <a class="subtle" href="{% url 'doc' 'services/lab' %}">more &raquo;</a>
-                        </span>
-                    {% else %}
-                        <strong>Lab Temporarily Closed</strong>
-                        <a class="subtle" href="{% url 'home' %}">more &raquo;</a>
-                    {% endif %}
+                    <strong>Vote for Student Tech Fund's renewal at ASUC!</strong>
```
Comment (ja5087) [inline ocfweb/templates/base.html:103]: maybe give the date? seems more actionable IMO

Maybe: Vote for the Student Tech Fund's Renewal during ASUC Elections on April 5-7!

File: ocfweb/templates/base.html [hunk 2]
```diff
@@ -97,30 +97,17 @@
...
+                    <span class="nowrap">
+                        OCF is funded by the Student Technology Fund, which will expire if not renewed.
```
Comment (ja5087) [inline ocfweb/templates/base.html:105]: nit: The OCF

File: ocfweb/docs/docs/communications/stf-renewal.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+[[!meta title="Upcoming Student Technology Fund renewal voting"]]
```
Comment (ja5087) [inline ocfweb/docs/docs/communications/stf-renewal.md:1]: Upcoming Student Technology Fund Renewal Vote 

or at least capitalize all the words

**Review by ja5087 — APPROVED** [review]
looks good to me. I'll request @snarain01 and @encadyma to give this a final pass.

**Comment by snarain01** [comment]
Just read through the changes and descriptions - this looks great! I can't think of other things to add aside from what jaw already said. Thank you for getting this done really quickly rjz!

**Pull request merged**